samanthas_telegram_bot/api_queries/send_get.py

```diff
@@ -0,0 +1,293 @@
+                "student_age_ranges": user_data.teacher_student_age_range_ids,
+                "teaching_languages_and_levels": user_data.language_and_level_ids,
+            },
+        )
+    if r.status_code == httpx.codes.CREATED:
+        await send_to_admin_group(
+            bot=update.get_bot(),
+            text=(
+                f"New adult teacher: [{user_data.first_name} {user_data.last_name}]"
+                f"({API_URL_PREFIX}/teacher/{personal_info_id})"
+            ),
+            parse_mode=ParseMode.MARKDOWN_V2,
+        )
+        logger.info(f"Chat {user_data.chat_id}: Created adult teacher")
+        return True
+
+    await log_and_report(
+        text=(
+            f"Chat {user_data.chat_id}: Failed to create adult teacher "
+            f"(code {r.status_code}, {r.content})"
+        ),
+        logger=logger,
+        level="error",
+        bot=update.get_bot(),
+        parse_mode=None,
+    )
+    return False
+
+
+async def send_teacher_under_18_info(update: Update, user_data: UserData) -> bool:
+    """Sends a POST request to create a teacher under 18 years old."""
+
+    personal_info_id = await _send_personal_info_get_id(user_data=user_data, bot=update.get_bot())
+
+    async with httpx.AsyncClient() as client:
+        r = await client.post(
+            f"{API_URL_PREFIX}/teachers_under_18/",
+            data={
+                "personal_info": personal_info_id,
+                "comment": user_data.comment,
+                "status_since": _format_status_since(update),
+                "status": STATUS_AT_CREATION_TEACHER_UNDER_18,
+                "can_host_speaking_club": user_data.teacher_can_host_speaking_club,
+                "has_hosted_speaking_club": False,
+                "is_validated": False,
+                "non_teaching_help_provided_comment": user_data.teacher_additional_skills_comment,
+                "teaching_languages_and_levels": user_data.language_and_level_ids,
+            },
+        )
+    if r.status_code == httpx.codes.CREATED:
+        await send_to_admin_group(
+            bot=update.get_bot(),
+            text=(
+                f"New *young* teacher: [{user_data.first_name} {user_data.last_name}]"
+                f"({API_URL_PREFIX}/teacherunder18/{personal_info_id})"
+            ),
+            parse_mode=ParseMode.MARKDOWN_V2,
+        )
+        logger.info(f"Chat {user_data.chat_id}: Created young teacher")
+        return True
+    await log_and_report(
+        text=(
+            f"Chat {user_data.chat_id}: Failed to create young teacher "
+            f"(code {r.status_code}, {r.content})"
+        ),
+        logger=logger,
+        level="error",
+        bot=update.get_bot(),
+        parse_mode=None,
+    )
+    return False
+
+
+def _format_status_since(update: Update) -> str:
+    """Converts a datetime object into suitable format for `status_since` field."""
+    # TODO can/should backend do this?
```

Comment:
It should be parsed same on the backend, I think, but not sure. I remember some problems in our backend tests with this datetime formatting, need to be checked.

Reply:
OK, leaving TODO for now